### ISSUE TITLE
Fetch version by specific url

### ISSUE THREAD

**Issue by Jnesselr** [cloudmatrix/esky#61]
Judging by how esky works, it should be possible to give it a specific URL to update to, even if it is a different app entirely. I'd like to do this because of the way that app I'm working with works. I wanted the patches to be stored on Amazon S3. When the client contacts the server, the server will mention that it can update the client if certain conditions are met. It also would give it the amazon S3 url at this point.

The current workaround I can see is to manually download the file from S3, and just point esky to a local directory, but I'm not sure if pointing it to a local directory is supported.

So basically, feature request.

**Comment by timeyyy** [comment]
Just merged a pull request to grab li.ks from s3 buckets
